src/components/App/App.js

```diff
@@ -5,11 +5,13 @@ import Header from '../Header/Header';
 import PrivateRoute from '../Utils/PrivateRoute';
 import PrivateRouteShelter from '../Utils/PrivateRouteShelter';
```

Comment:
Can we combine `PrivateRoute` and `PrivateRouteShelter`  in one file?

Reply:
Probably yeah. I didn't know about props before this PR. Let me try to figure something out!

Reply:
Just realize you meant to literally just move the function lol oops. Oh well, I added some logic to differentiate between regular and shelter routes. Let me know if you think your solutions is better

Reply:
Your change looks really good! Thank you!